static/src/javascripts/projects/common/modules/preferences/main.js

```diff
@@ -0,0 +1,69 @@
+define([
+    'react',
```

Comment:
do we really need react to do this? is react being used anywhere else that's visible?